pkgs/development/python-modules/dm-tree/default.nix

```diff
@@ -64,7 +56,7 @@ buildPythonPackage rec {
 
   build-system = [ setuptools ];
 
-  nativeCheckInputs = [
+  dependencies = [
```

Comment:
these were previously only test inputs and not propagated. why do we need to propagate them now?

Reply:
Huh weird... I created https://github.com/google-deepmind/tree/issues/127 to track. Let's link to that in a comment and proceed?